.travis.yml

```diff
@@ -33,6 +33,7 @@ env:
     - PHP_VERSION="7.1"
     - STACK_VERSION="2.1.1"
     - YARN_VERSION="1.16.0"
+    - RUST_VERSION="1.35.0"
```

Comment:
Nit: We generally try to keep this list sorted alphabetically for easier lookup.